### DISCUSSION TITLE
Syntax of style's squelets

### DISCUSSION THREAD

**Discussion by livier** [getpelican/pelican#3434] [Q&A]
Where can I get documentation about the syntax of theses king of things ?
```
{% for article in articles %}        
{% if loop.index == 1 %}

{{ article.content }}
```
And is there a way to order some articles in the loop to apear on top of the first page index.html

**Comment by guillermodotn** [comment]
The code you've provided is written in `Jinja2`, a templating engine for Python that allows you to embed dynamic expressions and control structures within your templates. You can find the [template documentation here](https://jinja.palletsprojects.com/en/stable/templates/).  

Regarding the order of display, you can use Jinja2's filters, such as [`sort()`](https://jinja.palletsprojects.com/en/stable/templates/#jinja-filters.sort), to order the articles before iterating through them.
